osu.Desktop/OsuGameDesktop.cs

```diff
@@ -137,6 +137,9 @@ protected override void LoadComplete()
             if (RuntimeInfo.OS == RuntimeInfo.Platform.Windows)
                 LoadComponentAsync(new GameplayWinKeyBlocker(), Add);
 
+            if (OperatingSystem.IsWindows() && IsDeployedBuild)
+                LoadComponentAsync(new WindowsAssociationManager(), Add);
```

Comment:
I also don't think that this is right. This shouldn't need to run on every game startup. It should be hooked into squirrel install flows. At the _most_ there should be a button similar stable to manually fix associations when they break, but the game shouldn't be trying to write the same thing to registry every launch and then re-raise notifications about changed associations when nothing really changed.

Reply:
Agreed. I'll look into making this static and only running once. (Or maybe on every update, in case the icons, etc. are changed.)